docs/getting-started-guides/locally.md

```diff
@@ -82,7 +82,7 @@ cluster/kubectl.sh create -f docs/user-guide/pod.yaml
 
 Congratulations!
 
-### Troubleshooting
+### FAQs
```

Comment:
Changed the title since "I changed Kubernetes code, how do I run it?" isn't really troubleshooting.